cmd/influxd/run.go

```diff
@@ -218,6 +222,10 @@ func openServer(path string, u *url.URL, b *messaging.Broker, initializing bool,
 			joinServer(s, u, joinURLs)
 			openServerClient(s, joinURLs)
 		}
+	} else if !configExists {
+		// We are spining up an server that has no config,
+		// but already has an initialized data directory
+		openDefaultServer(s, b)
```

Comment:
You shouldn't need to duplicate the replica and server creation. If there's no config then should just need to set `joinURLs` to `[]*url.URL{b.URL}` and call `openServerClient(s, joinURLs)`.